volume-providers/vpc/provider/list_volumes.go

```diff
@@ -43,6 +54,9 @@ func (vpcs *VPCSession) ListVolumes(limit int, start string, tags map[string]str
 	})
 
 	if err != nil {
+		if strings.Contains(err.Error(), "start parameter is not valid") {
```

Comment:
can be check the error code instead matching some string, if not possible then pl create a const value for this string

Reply:
Added string as const.